translate/uz.json

```
@@ -90,7 +90,7 @@
     "Chortoq": 35,
     "Chust": 33
   },
-  "chooseRegion": "Shaxaringizni belgilang",
+  "chooseRegion": "Shahringizni belgilang",
```

```suggestion
  "chooseRegion": "Mintaqani belgilang",
```
Chunki hammasi regionladgi hammasi shahar emas manimcha